backend/src/plugins/Automod/triggers/matchMimeType.ts

```diff
@@ -0,0 +1,79 @@
+import { automodTrigger } from "../helpers";
+import * as t from "io-ts";
+import { asSingleLine, messageSummary, verboseChannelMention } from "src/utils";
```

Comment:
```suggestion
import { asSingleLine, messageSummary, verboseChannelMention } from "../../../utils";
```

Reply:
Yeah I don't know how that happened, must've been a result of Visual Studio Code messing with my imports again. I'll fix it, thanks.